pep-0646.rst

```diff
@@ -307,19 +307,55 @@ length, and the type parameters themselves must be identical):
   pointwise_multiply(x, y)  # Error
   pointwise_multiply(x, z)  # Error
 
-Multiple Type Variable Tuples: Not Allowed
-''''''''''''''''''''''''''''''''''''''''''
+Multiple Type Variable Tuples Can Cause Ambiguity
+'''''''''''''''''''''''''''''''''''''''''''''''''
 
-As of this PEP, only a single type variable tuple may appear in a type parameter list:
```

Comment:
A very minimal change is to change this to:

As of this PEP, only a single type variable tuple may appear among a generic class's type parameters: